data/fixtures/scopes/zig/name.assignment.scope

```diff
@@ -0,0 +1,24 @@
+const foo = "bar";
```

Comment:
The assignment scope should be assigning to an already existing variable. Please rename this file to `name.variable`.